_data/hosting_providers.json

```
@@ -1,4 +1,14 @@
 [
+  {
+    "name": "Snel.com",
+    "link": "https://www.snel.com - Your friendly hosting provider",
```

This needs to be just the URL, since it forms the link in the list:

```suggestion
    "link": "https://www.snel.com",
```